<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewEntry</class>
 <widget class="QDialog" name="NewEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neuen Eintrag erstellen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelError">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineName"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Host:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineHost"/>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>SSH-Port:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineSshPort"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>User:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineUser"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Lokaler-Port:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineLocPort"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Remote-IP:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineExtIP"/>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Remote-Port:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineExtPort"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
